### PR TITLE
Fix #2498: Add 1.16 translations.

### LINKED ISSUE #2498: Add 1.16 translations
- [x] push new strings
- [x] pull translations

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

Found one duplicated string key `AboutBraveShields`, made a followup ticket for it https://github.com/brave/brave-ios/issues/2511

This pull request fixes #2498 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: BraveShared/de.lproj/BraveShared.strings [hunk 1]
```diff
@@ -301,6 +301,9 @@
 /* DuckDuckGo callout title. */
 "DDGCalloutTitle" = "Private Suche über DuckDuckGo?";
 
+/* Name for default Brave theme. */
+"defaultThemeName" = "Brave-Standarddesign";
+
```
Comment (kylehickinson) [inline BraveShared/de.lproj/BraveShared.strings:306]: This needed?

File: BraveShared/de.lproj/BraveShared.strings [hunk 2]
```diff
@@ -1174,6 +1189,9 @@
 /* Selection to color/style the user interface with a light theme. */
 "ThemesLightOption" = "Hell";
 
+/* Name for app theme settings */
+"themesSettings" = "Designs";
+
```
Comment (kylehickinson) [inline BraveShared/de.lproj/BraveShared.strings:1194]: Same here

File: BraveShared/de.lproj/BraveShared.strings (hunk 1 above)
Comment (kylehickinson) [reply BraveShared/de.lproj/BraveShared.strings:306]: Ah it looks like it's in every language. Nevermind

**Review by kylehickinson — APPROVED** [review]

File: BraveShared/de.lproj/BraveShared.strings (hunk 1 above)
Comment (iccub) [reply BraveShared/de.lproj/BraveShared.strings:306]: this German translations looks funny :D it's for custom home page/themes setting

Comment (kylehickinson) [reply BraveShared/de.lproj/BraveShared.strings:306]: Haha yeah the key made me think we accidentally snuck in a static file name dictionary key or something into the strings file 😆

**Pull request merged**